pkgs/tools/misc/entr/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "entr-${version}";
+  version = "3.2";
+
+  src = fetchurl {
+    url = "http://entrproject.org/code/${name}.tar.gz";
+    sha256 = "0ikigpfzyjmr8j6snwlvxzqamrjbhlv78m8w1h0h7kzczc5f1vmi";
+  };
+
+  builder = ./builder.sh;
```

Comment:
@cheecheeo: `make test` should be run in a separate `checkPhase`, so it can be easily controlled with `doCheck`.

See http://nixos.org/releases/nixpkgs/nixpkgs-0.12/manual/#ssec-stdenv-phases.